### DISCUSSION TITLE
TEN Projects

### DISCUSSION THREAD

**Discussion by stud01000** [chreden/trview#1135] [Ideas]
Is it possible in the future to open TOMBEngine levels?

**Comment by Trxyebeep** [comment]
bit of info: TEN is an engine still in active development and the level format will constantly change as more updates come.
Not to mention, its level format (as of now) already greatly differs from the original. Not worth the effort imo.

**Comment by chreden** [comment]
Probably not soon, but I'll bear it in mind as I assume more and more levels are going to be made using that format. I didn't even realise it had its own format, so it's good to know that at least.
